styles/main.css

```diff
@@ -70,25 +71,31 @@ header>nav>ul {
     padding: 2em;
     margin-bottom: 3em;
     border: 1px solid rgba(0, 0, 0, 0.3);
+    border-top: none;
     background: #ffecc1;
+    box-shadow: 0 0 4px rgba(0,0,0,0.2);
 }
 
 figure {
-    box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.5);
     text-align: center;
 }
 
 figure img {
     width: 80%;
+    box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.5);
 }
 
 footer hr {
-    width: 50%;
+    width: 50vw;

```

Comment:
Use the vw unit to have widths relative to the browser viewport.  The % unit is relative to the parent element